lib/instances/TextLabel.lua

```diff
@@ -0,0 +1,48 @@
+local Color3 = import("../types/Color3")
+local GuiObject = import("./GuiObject")
+local InstanceProperty = import("../InstanceProperty")
+local TextLabel = GuiObject:extend("TextLabel")
+
+TextLabel.properties.Font = InstanceProperty.normal({
+	getDefault = function()
+		return 0
+	end,
+})
+
+TextLabel.properties.Text = InstanceProperty.normal({
+	getDefault = function()
+		return ""
+	end,
+})
+
+TextLabel.properties.TextColor3 = InstanceProperty.normal({
+	getDefault = function()
+		return Color3.new()
+	end,
+})
+
+TextLabel.properties.TextSize = InstanceProperty.normal({
+	getDefault = function()
+		return 12
+	end,
+})
+
+TextLabel.properties.TextWrap = InstanceProperty.normal({
+	getDefault = function()
+		return false
+	end,
+})
+
+TextLabel.properties.TextXAlignment = InstanceProperty.normal({
+	getDefault = function()
+		return 0
```

Comment:
Should be an Enum

Reply:
Enum.TextXAlignment.Left maps to 0 though, is it ok to just return 0 here?

Reply:
Sure, but it's more intuitive for an enum.

Reply:
From a code style / maintainability PoV, having `Enum.TextXAlignment.Left` here instead of `0` both looks cleaner and is more bulletproof against future changes.